koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -738,18 +672,16 @@ class BuildContainerTask(BaseContainerTask):
         "minItems": 3
     }
 
-    def __init__(self, id, method, params, session, options, workdir=None, demux=True):
+    def __init__(self, id, method, params, session, options, workdir=None):
         # pylint: disable=redefined-builtin
         BaseContainerTask.__init__(self, id, method, params, session, options,
                                    workdir)
-        self.demux = demux
         self.event_id = None
 
     def createContainer(self, src=None, target_info=None, arches=None,
                         scratch=None, isolated=None, yum_repourls=None,
-                        branch=None, push_url=None, koji_parent_build=None,
-                        release=None, flatpak=False, signing_intent=None,
-                        compose_ids=None, skip_build=False, triggered_after_koji_task=None,
+                        branch=None, koji_parent_build=None, release=None,
+                        flatpak=False, signing_intent=None, compose_ids=None,
```

Comment:
I guess it's inevitable that we will change the public API for OSBS 2.0, but is it OK to just remove the params like this?

Reply:
skip_build & triggered_after are related to autorebuilds

push_url wasn't even used

Reply:
Yeah, probably no way around that, I was just wondering if we had some process for doing breaking API changes

Reply:
flatpak?

Reply:
usually we first deprecate stuff and then remove, but this is new osbs 2